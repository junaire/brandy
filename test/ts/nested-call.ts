var y = 1n + getTwo();
console.log(y);

function getTwo(): bigint {
    return 2n;
}
