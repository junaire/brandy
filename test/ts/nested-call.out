@main {
  v0: int = const 1;
  v1: int = call @getTwo;
  v2: int = add v0 v1;
  y: int = id v2;
  v3: int = id y;
  print v3;
  v4: int = const 0;
}
@getTwo: int {
  v0: int = const 2;
  ret v0;
}
